packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -9,9 +9,9 @@ import { hexValue } from 'ethers/lib/utils'
 import { BigNumber } from 'ethers'
 import * as Sentry from '@sentry/react'
 
-import EthereumLogo from '../assets/EthereumLogo.png'
+import EthereumLogo from '../assets/EthereumLogo.webp'
 import ArbitrumOneLogo from '../assets/ArbitrumOneLogo.svg'
```

Comment:
Any reason only this is kept in svg?

Reply:
the goal is to optimise for speed and reduce load
svg generally is the most optimised format for vector image if available
converting this image to webp will increase its size and reduce image quality